2.form.html

```diff
@@ -0,0 +1,78 @@
+﻿<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="utf-8" />
+    <title></title>
+</head>
+<body>
+    <h1>Payment Form</h1>
+    <h4>Requierd Fields are followed by .*.</h4>
+    <div style="line-height: 1cm;">
+        <h>Contact Information</h>
+        <form action="" method="GET">
+            <fieldset>
+                <fieldset>
+                    <legend>Choose</legend>
+                    <div>
+                        <label>Are you Teacher or Student ?</label>
+                    </div>
+                    <div>
+                        <label for="Studenttype">Student</label>
+                        <input name="Student" type="radio" value="Student">
+                    </div>
+                    <div>
+                        <label for="Teachertype">Teacher</label>
+                        <input name="Teacher" type="radio" value="Teacher">
+                    </div>
+                </fieldset>
+                <div>
+                    <label for="username">Name .*. </label>
+                    <input name="username" required="" type="text" placeholder="Username">
+                </div>
+                <div>
+                    <label for="password">Password .*. </label>
+                    <input min="8" name="password" required="" type="password">
```

Comment:
required attr in boolean. Meaning its either there (true). Or its not there (false). required="" will never have a value so lets remove =""